src/l2_packet/l2_packet_zephyr.c

```diff
@@ -169,8 +169,7 @@ l2_packet_init(const char *ifname, const u8 *own_addr, unsigned short protocol,
 	ll.sll_protocol = htons(protocol);
 
 	// FIXME: This should skip bind_default to ETH_P_ALL, but its not.
```

Comment:
Is this FIXME still valid after this change?

Reply:
Yes, this code doesn't change any functional behaviour, just a unnecessary NULL check